package.json

```diff
@@ -1,15 +1,31 @@
 {
-  "name": "rsk-transaction-helper",
-  "version": "3.2.0",
-  "description": "Utilitary library to send transactions on RSK using Web3 and Ethereum-js",
+  "name": "rootstock-transaction-helper",
+  "version": "4.0.0",
+  "description": "Utilitary library to send transactions on Rootstock using web3 and ethereum-js",
   "main": "index.js",
   "scripts": {
     "test": "mocha ./tests",
     "coverage": "nyc npm run test",
     "sample": "node ./samples/send-transaction.js"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rsksmart/rsk-transaction-helper.git"
+  },
+  "bugs": {
+    "url": "https://github.com/rsksmart/rsk-transaction-helper/issues"
+  },
+  "homepage": "https://github.com/rsksmart/rsk-transaction-helper#readme",
```

Comment:
The urls Should Be *rootstock-transaction-helper

Reply:
Good catch. Thanks!